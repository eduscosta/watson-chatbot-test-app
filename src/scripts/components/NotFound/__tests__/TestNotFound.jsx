/* eslint-env jest */

import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../NotFound';

describe('<NotFound />', () => {
  const tree = shallow(<NotFound />);

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
