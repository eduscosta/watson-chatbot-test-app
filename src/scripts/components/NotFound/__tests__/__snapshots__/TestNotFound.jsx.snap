// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders correctly 1`] = `
<article
  className="main"
>
  <header>
    <h1>
      404.
    </h1>
  </header>
  <p>
    <Link
      replace={false}
      to="/"
    >
      Take me home.
    </Link>
  </p>
</article>
`;
